src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -144,12 +151,22 @@ await functionMetadata
                 await FileUtility.WriteAsync(dataFilePath, functionMetadata.TestData);
             }
 
+            var metadata = (await _hostFunctionMetadataProvider.GetFunctionMetadataAsync(_languageWorkerOptions.CurrentValue.WorkerConfigs, configChanged))
+                .FirstOrDefault(metadata => Utility.FunctionNamesMatch(metadata.Name, name));
+
+            var success = false;
```

Comment:
Can we rename this to something more descriptive, what is successful here?

Reply:
done